src/plugins/SysML/SysMLExporter/SysMLExporter.js

```diff
@@ -135,7 +138,16 @@ define(['plugin/PluginConfig',
             isRqtParent = isPackage || self.isMetaTypeOf(parentBaseClass, self.META.RequirementDiagram),
             isRqtDiagram = isRqtParent && (isRequirement),
             isReq2Req = self.isMetaTypeOf(baseClass, self.META.Req2Req),
-            afterConnAdded;
+            afterConnAdded,
+			/** Block Definition Diagram **/
+			isBlock = self.isMetaTypeOf(baseClass, self.META.Block),
+			isBDDReq = self.isMetaTypeOf(parentBaseClass, self.META.BlockDefinitionDiagram),
+			isBDDParent = isPackage || self.isMetaTypeOf(parentBaseClass, self.META.BlockDefinitionDiagram),
+			isBlockAssociations = self.isMetaTypeOf(baseClass, self.META.Associations),
+			isBDDDiagram = isBDDParent && (isBDDReq || isBlockAssociations);
+			
```

Comment:
Indentation fixed. I think maybe git shows it in deeper indent to show the difference.